### ISSUE TITLE
Gantt Charts stopped working, need some help here =)

### ISSUE THREAD

**Issue by azkdev** [neuronetio/gantt-elastic#23]
Hi, after last update 10h ago, Gantt Component stopped working with the following error:

[Screenshot here](https://imgur.com/a/jQs4W8y)

Can you explain why this happened ? Thanks in advance.
Below is a little piece of code, how I initialize Gantt.
And why I can't get gantt instance via event 'gantt-elastic-ready' ? =)
Sorry for my english :P

```
const gantt = new Vue({
    components: {
      'gantt-header': Header,
      'gantt-elastic': GanttElastic
    },
    data: {
      tasks: [],
      options,
      showHeader: true
    },
    computed: {
      gellastic() {
        return this.$refs.gellastic;
      },
      gellasticheader() {
        return this.$refs.gellasticheader;
      }
    },
    watch: {
      tasks() {
        setTimeout(() => {instance.setHeight(this.gellastic.state.height)}, 100);
      }
    }
  }).$mount(ganttWrapper);
```

**Comment by neuronetio** [comment]
yap, something bad happened
I'm working on it, thanks for report!

**Comment by azkdev** [comment]
Thank you for operative answer,
Leave a feedback when it will be done, thank you in advance =)

Maybe there is a problem in ligatures? Null Array

**Comment by neuronetio** [comment]
fixed in v0.8.1

**Comment by azkdev** [comment]
It works, thank you dude ;)
Closed
